db/migrate/20230719211602_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
```

Comment:
- [ ] Please add foreign keys to the user and post tables as mentioned in the requirements. It can look something like this:
```ruby
    t.references :author, foreign_key: { to_table: 'users' } 
    t.references :post, foreign_key: true
```